### DISCUSSION TITLE
Conditional card / Add support for attributes

### DISCUSSION THREAD

**Discussion by Ulrar** [home-assistant/frontend#11855] [Dashboard] [answered]
Hi,

The conditional card does not support attributes, so the workaround is to extract the attributes needed using a template sensor.
It'd be really nice not to have to do that.

Looks like someone already had done the work but the PR was closed stale a while back, maybe it could get reviewed still ? https://github.com/home-assistant/home-assistant.io/pull/15648

Thanks

**Comment by ZephireNZ** [comment, marked as answer]
FYI I have opened #12867 which would resolve this using a template
